Prac_05/word_occurrences.py

```diff
@@ -0,0 +1,13 @@
+words_to_count = {}
```

Comment:
Be careful with plurals. This implies that each key is "words" instead of "word".